.github/workflows/ci.yml

```diff
@@ -488,12 +488,9 @@ jobs:
       uses: actions/setup-python@v5.0.0
       with:
         python-version: ${{ matrix.PYTHON }}
-    - uses: actions-rs/toolchain@v1
+    - uses: dtolnay/rust-toolchain@master
```

Comment:
Would it be safer to stick on some tag or commit?

Reply:
Unfortunately this is the way to use the exact version of the rust toolchain from the matrix. See the documentation of the dtolnay/rust-toolchain action.